### PR TITLE
Creation of Global Markdown Formatting and Linting Standards Documentation

### PR DESCRIPTION
**Pull request by damienjburks**
## Summary

This is the initial PR for creating the global markdown formatting and lining standards for the entire project. There is a new folder within the governance section of our project that will also house the release strategy and WG formatting standards (very soon). 

The information listed here correlates with what has already been done with the GitHub Action PR: #223 

Let me know if you have any questions, comments, or concerns.

## Linked Issues/PRs

- #209 
- #223

### CODE REVIEW

File: docs/governance/content-standards-and-practices/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Table of Contents
+
+- [Purpose](#purpose)
+- [How to Use These Guidelines](#how-to-use-these-guidelines)
+- [Current Guidelines and Standards](#current-guidelines-and-standards)
+- [Feedback and Contribution Disclaimer](#feedback-and-contribution-disclaimer)
+
+## Purpose
+
+This directory will contain the conten development standards and practices, where you can find all the necessary guidelines and standards for formatting, linting, and releasing content within our projects. The purpose of this folder is to provide clear, accessible guidelines that enhance our workflows and maintain high standards of quality. Adherence to these guidelines is crucial for maintaining the integrity and consistency of our content. All [WG]'s are required to follow these guidelines.
```
Comment (sshiells-scottlogic) [inline docs/governance/content-standards-and-practices/README.md:12]: Change "conten" to "content"

File: docs/governance/content-standards-and-practices/README.md
Comment (sshiells-scottlogic) [inline docs/governance/content-standards-and-practices/README.md:1]: Should this be a "policy" contained in "community-policies"?

We currently have "comminity-guidelines" which are strong suggestions on what WGs should do, and "community-policies" which are things WGs must adhere to?

The wording here suggests that this should be a policy to me? If you agree, then maybe we need to change the use of "guidelines" in this text?

File: docs/governance/content-standards-and-practices/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## How to Use These Guidelines
+
+1. **Review Guidelines Before Starting Your Project:** Familiarize yourself with the formatting and linting standards to ensure your work aligns from the outset.
+2. **Apply the Standards Consistently:** Please ensure that you have installed the necessary tools and enabled the automation locally (if applicable) to ensure you are following the standards consistently.
+3. **Check Compliance Regularly:** Regularly review your work against these standards to ensure ongoing compliance, especially before merges and releases. We have automated processes in place that will check your MD files prior to being merged into the main branch.
```
Comment (sshiells-scottlogic) [inline docs/governance/content-standards-and-practices/README.md:18]: Should we use "markdown" instead of "MD" for clarity?

**Review by sshiells-scottlogic — COMMENTED** [review]
Looks good. I left a question about where this should belong, and also a couple of minor comments.

File: docs/governance/content-standards-and-practices/README.md
Comment (damienjburks) [reply docs/governance/content-standards-and-practices/README.md:1]: I think that's a great idea. I just updated the wording a bit and moved everything into the community-policies directory. 😄

**Comment by damienjburks** [comment]
@sshiells-scottlogic addressed all of your comments, let me know if there is anything else you think I need to address.

**Review by sshiells-scottlogic — APPROVED** [review]

**Comment by damienjburks** [comment]
@eddie-knight please review and let me know your thoughts on this.

**Comment by eddie-knight** [comment]
Hey @damienjburks — the only hangup here is that we already wrote "policies may be created or modified by a vote of the SC."

If you want to merge this in as a policy, it'll need to wait in PR mode until we get 4 votes from steerco members. We can send out an email requesting folks come take a stance, or we can merge this as a guideline and then call for it to be promoted on the next SC call. I prefer the second option, if that's okay with you.

**Review by eddie-knight — CHANGES_REQUESTED** [review]
Hey @damienjburks — the only hangup here is that we already wrote "policies may be created or modified by a vote of the SC."

If you want to merge this in as a policy, it'll need to wait in PR mode until we get 4 votes from steerco members. We can send out an email requesting folks come take a stance, or we can merge this as a guideline and then call for it to be promoted on the next SC call. 

I prefer the second option, if that's okay with you. The only change would be to move this into the other directory.

**Comment by damienjburks** [comment]
@eddie-knight that is fine with me. Shall I revert the wording to guidelines then?

**Comment by eddie-knight** [comment]
Yep, good shout.

**Comment by damienjburks** [comment]
@eddie-knight everything has been updated. let me know if this works for you.

File: markdown/formatting-guidelines.md
Comment (eddie-knight) [inline markdown/formatting-guidelines.md:1]: Looks like this is unintentional

Comment (damienjburks) [reply markdown/formatting-guidelines.md:1]: Fixed!

**Review by eddie-knight — APPROVED** [review]

**Pull request merged**